pkg/util/utils.go

```diff
@@ -97,8 +97,11 @@ func CreateCustomResourceClients(apiserver string, kubeconfig string, factories
 		if err != nil {
 			return nil, err
 		}
-		gvrString := GVRFromType(f.Name(), f.ExpectedType()).String()
-		customResourceClients[gvrString] = customResourceClient
+		gvr, err := GVRFromType(f.Name(), f.ExpectedType())
+		if err != nil {
+			return nil, err
+		}
+		customResourceClients[gvr.String()] = customResourceClient
```

Comment:
Here too?

Reply:
Also fixed in https://github.com/kubernetes/kube-state-metrics/pull/2553/commits/c3b1afe61c2a1b7335d4caf99f1ae228dfadbc18